rpcs3/Emu/Cell/Modules/libmixer.cpp

```diff
@@ -6,6 +6,13 @@
 #include "cellAudio.h"
 #include "libmixer.h"
 
+#if __GNUC__ >= 6
+#include <math.h>
+#define FABS(X) std::fabs(X)
```

Comment:
You can try and test against buildbots. It may require including <cmath> though (which is better than math.h anyway). Introducing preprocessor shenanigans is not desired.